.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+##### Description of Change
+<!--
```

Comment:
Are these comments going to show up in the PR template shown to the reporter?  (They don't show up in the rendered .md file on GitHub.)

Reply:
They will not show up, but will be visible when using the command line to submit a PR.